VD2901/AdminPanel
# Fix : remove extra code

**Close** #6 


**Summary**
  - This pull request addresses an issue with the image upload functionality where files larger than 5MB fail to upload, despite the 
    system being set to handle up to 10MB files. The fix ensures that the image upload feature now correctly handles files up to 
   10MB as intended, improving the user experience for those uploading high-resolution images.
  
 -  Additionally, it resolves a misconfiguration in the file size validation logic and updates the corresponding error messaging to 
     reflect the actual limits, ensuring consistency between system behavior and user expectations.
     
 **Testing Instructions**
  - Navigate to the Image Upload Form
  - Test with Small Files (< 5MB)
  - Test with Files Between 5MB and 10MB
  - Test with Large Files (> 10MB)
  - Verify Error Messaging
  - Test for Edge Cases
